docs/build-mvm/how-to/schedule_functions.md

```diff
@@ -1 +1,883 @@
+- Network env: `local`
+
+**TODO: when all updates are published, update to `kras` and the latest FCLI**
+
+## Create your first Spell: step-by-step
+
+First let's take a look at how to create a simple spell from the Fluence CLI template.
+
+1. Initialize a project with `fluence init -t minimal` 
+    
+    The spell library dependency should be added to the `fluence.yaml` configuration file automatically:
+    
+    ```yaml
+    # FCLI version: 0.13.5
+    #...
+    
+    aquaDependencies:
+      "@fluencelabs/aqua-lib": 0.9.1
+      "@fluencelabs/spell": 0.6.9
+    ```
+    
+2. Initialize a new spell template with `fluence spell new myFirstSpell`.
+    
+    This command modifies the `fluence.yaml` file, adding the spell to the list of the project’s spells and the default deal we deploy later.
+    
+    ```yaml
+    # FCLI version: 0.13.5 
+    deals:
+      dealName:
+        targetWorkers: 1
+        pricePerWorkerEpoch: 0.1
+        collateralPerWorker: 1
+        services: []
+        spells: [ myFirstSpell ]  # <-- added the spell to the deal
+    
+    spells:
+      myFirstSpell:   # <-- added the new spell to the list of project's spells 
+        get: src/spells/myFirstSpell # <-- the path to the spell directory 
+    ```
+    
+    This command will create a directory for the spell and its configuration in the `src/spells` directory:
+    
+    ```bash
+    $ tree src/spells
+    src/spells
+    └── myFirstSpell
+        ├── spell.aqua
+        └── spell.yaml
+    ```
+    
+    The path to the spell directory is also set in the `fluence.yaml` file and can be manually modified if needed.
+    
+3. The spell script is located in the `spell.aqua` file:
+    
+    ```aqua
+    -- the Fluence CLI version is 0.13.5
+    
+    aqua MyFirstSpell
+    
+    -- Note: spell main function must be exported
+    export spell
+    
+    import Spell from "@fluencelabs/spell/spell_service.aqua" -- (1)
+    
+    func spell(): -- (2)
+        Spell "myFirstSpell" -- (3)
+        Spell.store_log("Spell 'myFirstSpell' is working!") -- (4)
+    ```
+    
+    Let’s review the most important parts of the file:  
+    
+    1. Importing the (Spell Service API)[#how-to-resolve-your-spell-service] is required to use logging capabilities, private spell storage, etc. Note that you can implement spells without ever interacting with the Spell Service if you don’t need the features it provides.
+    2. The `spell` function here is an entry point to the spell’s execution. The main function has several features we explore in the [later](#inital-arguments) sections.
+        
+        You can modify the name of the main function to any valid name, but you also need to modify the name in the `spell.yaml` configuration file.
+        
+    3. To write logs in the [Spell Log Storage](#spell-logs-api), you need to resolve [link to the services resolution in aqua] the Spell Service that belongs to the executing spell using the Spell’s Name.
```

Comment:
Is/Will be there such a link?

Reply:
unless you create it, no. so linking into source is the safe way to go.